operator/docs/standard-operational-procedures/alert-loki-request-latency.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Alert - LokiRequestLatency"
+description: "The standard operational procedure for alert LokiRequestLatency"
+lead: ""
+date: 2022-06-21T08:48:45+00:00
+lastmod: 2022-06-21T08:48:45+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "standard-operational-procedures"
+weight: 100
+toc: true
+---
+
+## Impact
+
+A service(s) is affected by slow request responses.
+
+## Summary
+
+A service(s) is slower than expected at processing data.
+
+## Severity
+
+`Critical`
+
+## Access Required
+
+- Console access to the cluster
+- Edit access to the deployed operator and Loki namespace:
+  - OpenShift
+    - `openshift-logging` (LokiStack)
+    - `openshift-operators-redhat` (Loki Operator)
+
+## Steps
+
+- Check the logs of all the services
```

Comment:
What is the user looking for when they check the logs?  Checking all logs for all services is a huge query.